gopls/internal/test/marker/testdata/codeaction/extract_method.txt

```diff
@@ -180,39 +185,153 @@ func (b *B) LongList(ctx context.Context) (int, error) {
 	p3 := 1
 	return p1 + p2 + p3, ctx.Err() //@loc(B_LongList, re`return.*ctx\.Err\(\)`)
 }
+func (b *B) AddPWithB(ctx context.Context, tB *testing.B) (int, error) {
+	sum := b.x + b.y //@loc(B_AddPWithBStart, "s")
+	tB.Skip() 
+	return sum, ctx.Err() //@loc(B_AddPWithBEnd, re`$`)
+}
+
+func (b *B) LongListWithT(ctx context.Context, t *testing.T) (int, error) {
+	p1 := 1
+	p2 := 1
+	p3 := 1
+	p4 := p1 + p2 //@loc(B_LongListWithTStart, "p")
+	t.Skip()
+	return p4 + p3, ctx.Err() //@loc(B_LongListWithTEnd, re`$`)
+}
 -- @contextMeth1/context.go --
-@@ -17 +17 @@
+@@ -22 +22 @@
 -	return sum, ctx.Err() //@loc(B_AddP, re`return.*ctx\.Err\(\)`)
 +	return b.newMethod(ctx, sum) //@loc(B_AddP, re`return.*ctx\.Err\(\)`)
-@@ -20 +20,4 @@
+@@ -25 +25,4 @@
 +func (*B) newMethod(ctx context.Context, sum int) (int, error) {
 +	return sum, ctx.Err()
 +}
 +
 -- @contextMeth2/context.go --
-@@ -24 +24 @@
+@@ -29 +29 @@
 -	return p1 + p2 + p3, ctx.Err() //@loc(B_LongList, re`return.*ctx\.Err\(\)`)
 +	return b.newMethod(ctx, p1, p2, p3) //@loc(B_LongList, re`return.*ctx\.Err\(\)`)
-@@ -26 +26,4 @@
+@@ -31 +31,4 @@
 +
 +func (*B) newMethod(ctx context.Context, p1 int, p2 int, p3 int) (int, error) {
 +	return p1 + p2 + p3, ctx.Err()
 +}
 -- @contextFunc2/context.go --
-@@ -24 +24 @@
+@@ -29 +29 @@
 -	return p1 + p2 + p3, ctx.Err() //@loc(B_LongList, re`return.*ctx\.Err\(\)`)
 +	return newFunction(ctx, p1, p2, p3) //@loc(B_LongList, re`return.*ctx\.Err\(\)`)
-@@ -26 +26,4 @@
+@@ -31 +31,4 @@
 +
 +func newFunction(ctx context.Context, p1 int, p2 int, p3 int) (int, error) {
 +	return p1 + p2 + p3, ctx.Err()
 +}
 -- @contextFunc1/context.go --
-@@ -17 +17 @@
+@@ -22 +22 @@
 -	return sum, ctx.Err() //@loc(B_AddP, re`return.*ctx\.Err\(\)`)
 +	return newFunction(ctx, sum) //@loc(B_AddP, re`return.*ctx\.Err\(\)`)
-@@ -20 +20,4 @@
+@@ -25 +25,4 @@
 +func newFunction(ctx context.Context, sum int) (int, error) {
 +	return sum, ctx.Err()
 +}
 +
+-- @contextFunc2T/context.go --
+package extract
+
+import (
+	"context"
+	"testing"
+)
+
+//@codeactionedit(B_AddP, "refactor.extract", contextMeth1, "Extract method")
+//@codeactionedit(B_AddP, "refactor.extract", contextFunc1, "Extract function")
+//@codeactionedit(B_LongList, "refactor.extract", contextMeth2, "Extract method")
+//@codeactionedit(B_LongList, "refactor.extract", contextFunc2, "Extract function")
+//@codeaction(B_AddPWithBStart, B_AddPWithBEnd, "refactor.extract", contextFunc1T, "Extract function")
```

Comment:
I wanted to use codeactionedit, but I don't see a way to use it for a multi-line selection